css/types/color.json

```diff
@@ -137,7 +138,9 @@
                 "opera": "mirror",
                 "opera_android": "mirror",
                 "safari": {
-                  "version_added": "16.4"
+                  "version_added": "16.4",
+                  "partial_implementation": true,
+                  "notes": "Implementation based on older spec version. As a result, channel value calculations require values to specified as percentages with units (<code>%</code>)."
```

Comment:
I am still a bit cautious about explaining/documenting the older version of the spec.

I am concerned that authors will fixate on this and start manually crafting variants for each specific engine.

If such patterns emerge it can take a very long time for them to go away.

--------

Maybe this is a clearer message:
> Calculations with channel values do not work correctly.

It is less precise but it doesn't nudge authors towards writing engine specific CSS. It only states what doesn't work as specified.

Reply:
I think it is still worth explaining how they are incorrect. To not do so just seems like withholding information, and people will want to know how they can do testing in Safari.

Besides which, the demos I include in the accompanying docs PR show syntax examples of how to get the code working in both Chromium and WebKit at the same time. I need to include something to show how to do this. At a later date, once they have both supported the correct syntax for a little while, we can update the content to get rid of such code.

I do however think I should update these lines to make it explicit that this information is incorrect. Start the second line in each case with "Calculations with channel values do not work correctly..." but provide an explanation of how.

Reply:
I think the updated wording is very clear, thank you

Reply:
Lovely, thank you for the review, @romainmenke!